### DISCUSSION TITLE
Unable to build the Resume section

### DISCUSSION THREAD

**Discussion by sachinkmohan** [thetizzo/thetizzo.github.io#12] [General]
Hi Jason! Thanks for the wonderful blog on building resume! 

I understood the part where you have to create `resume.yml` under `_data` folder.

I don't understand, where to put this code?

```
{% for job in resume.jobs %}
  <div>
    <h3>
      <a href={{ job.company.website }} target="_blank">{{ job.company.name }}</a>
    </h3>

    {% for position in job.positions %}
      <div>
        <span>{{ position.title }}</span>
        <span>({{ position.duration }})</span>
      </div>
    {% endfor %}

    <p>{{ job.description }}</p>

    <ul>
      {% for accomplishment in job.accomplishments %}
        <li>{{ accomplishment }}</li>
      {% endfor %}
    </ul>
  </div>
{% endfor %}
```

Could you please help?

**Comment by sachinkmohan** [comment]
I was able to fix this part. But unable to generate a resume using the javascript. 

I created a resume with `layout: page`, so that I could see that under a new page.

```
---
layout: page
title: Resume
permalink: /resume/
resume: true
---

<div class="hidden-screen-lg mt-15">
	{% if layout.resume %}
	<a
		class="button button-sidebar"
		title="Download my resume"
		data-toggle="download-pdf"
	>
		<i class="fas fa-download"></i>
		Resume PDF
	</a>
	{% endif %}
</div>

{% include resume/resume.html %}

<div hidden id="resumeJson">{{ site.data.resume | jsonify }}</div>
```

Also the javascript(`resume.js`) was copied to the folder `/assets/javascripts/`

Finally in `default.html`, I wrote the below

```
    {% if layout.resume %}
    <script defer src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
    <script defer src="/vendor/pdfmake/pdfmake.min.js"></script>
    <script defer src="/vendor/pdfmake/vfs_fonts.js"></script>
    <script defer src="/assets/javascripts/resume.js"></script>
    <script defer src="/assets/javascripts/collapse.js"></script>
    {% endif %}
```
Am I missing something? Could you please help.

**Reply by thetizzo** [reply]
Hey, sorry for the delayed response but I just saw this post!

I hope you figured this out by now 😄 but I think this all looks right to me. Do you have any errors in the console when you click the button?

I would check to make sure the [click handler](https://github.com/thetizzo/thetizzo.github.io/commit/bcda711d0d864595d49add2ece1ca2442ab516f3#diff-c80df7b21c456ab1b8339f24f3f4df52f253b4eb7b55bc14a95ee01c713edc78L177-L180) is firing correctly and calling the `generateResume` function like you would expect.

If that is working properly I would make sure that the resume data in the hidden div is getting rendered correctly with valid JSON and that it's getting parsed correctly by the JavaScript [here](https://github.com/thetizzo/thetizzo.github.io/commit/bcda711d0d864595d49add2ece1ca2442ab516f3#diff-c80df7b21c456ab1b8339f24f3f4df52f253b4eb7b55bc14a95ee01c713edc78L5)
